edx_proctoring/backends/tests/test_rest.py

```diff
@@ -3,19 +3,23 @@
 """
 from __future__ import absolute_import
 
-import json
+import responses
+from mock import patch
```

Comment:
This ordering seems odd, not sure why isort moved these.  The `mock` one I can understand if it was run under Python 3, but `responses` should be a 3rd-party package under all Python versions.